docs/GDPs/gdp-index.rst

```diff
@@ -23,6 +23,8 @@ Open GDPs (under consideration)
 .. toctree::
    :maxdepth: 1
 
+   gdp-0001-standalone-cli
```

Comment:
In the GDP 0 is stated that GDP files should be named just with the number, like `gdp-XXXX.rst`, but I also find useful this notation with a short topic description. Should we thus change the GDP to use a more readable notation like it's being used here: `gdp-XXXX-topic-description.rst`?

Reply:
Sure, do you think it would be ok to "sneak in" the change in this PR?

Reply:
I'd rather do it in a separate commit before merging this PR.